### DISCUSSION TITLE
Restoring configuration from archiso on a fresh-new install

### DISCUSSION THREAD

**Discussion by izmyname** [CyberShadow/aconfmgr#213] [Ideas]
If I understand correctly - restoring a system state on a fresh new install requires that install to be bootable and a user account created , so aconfmgr config could be placed under `~/.config` and applied. The question is - is it possible to bypass those steps and apply config from root account `/root/.config` from archiso `arch-chroot`?

**Comment by CyberShadow** [comment]
Sure, you can also specify the config location from the command line. You'll still need to chroot into the new system with `arch-chroot`.

**Reply by izmyname** [reply]
It's good to know! Thank you!

**Reply by izmyname** [reply]
Another question, if I may

If I declare a config in xx*.sh, for example

```
cat >"$(CreateFile /etc/modprobe.d/blacklist.conf)" <<EOF
blacklist uvcvideo
EOF
```
Do I need to add `IgnorePath '/etc/modprobe.d/blacklist.conf'` to my `10-ignore.sh` or aconfmgr handles this, somehow?

**Reply by CyberShadow** [reply]
This would create a file managed by `aconfmgr`, so there is no need to ignore it. Ignoring is useful for files that shouldn't be managed, e.g. log files.

**Reply by izmyname** [reply]
Got it, thank you for clarification
